### PR TITLE
Productivity suite - parallelize fetches

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

File: productivity-suite/piercing-library/src/piercing-gateway.ts [hunk 1]
```diff
@@ -100,14 +99,6 @@ export class PiercingGateway<Env> {
     this.fragmentConfigs.set(fragmentConfig.fragmentId, fragmentConfig);
   }
 
-  /*
-		We're assigning fetch to the object itself so that user can do:
-			`export default gateway;`
-		this shouldn't be necessary and we should be able to have fetch
-		as a normal method in the class, but that doesn't seem to
-		currently being recognized correctly:
-			https://chat.google.com/room/AAAADFUqSgo/njWMATzSHG4
```
Comment (dario-piotrowicz) [inline productivity-suite/piercing-library/src/piercing-gateway.ts:109]: this refers to a private gchat conversation so I think we can/should just exclude it (it is also not an issue related to the piercing strategy)

Comment (petebacondarwin) [reply productivity-suite/piercing-library/src/piercing-gateway.ts:109]: Actually, I think the main comment is a valid thing to keep here. Perhaps just remove the link to the gchat?

**Review by petebacondarwin — APPROVED** [review]

File: productivity-suite/piercing-library/src/piercing-gateway.ts [hunk 2]
```diff
@@ -3,7 +3,6 @@ import {
   piercingFragmentHostInlineScript,
 } from "./index";
 import { concatenateStreams, wrapStreamInText } from "./stream-utilities";
-// import qwikloader from "@builder.io/qwik/qwikloader.debug.js?raw";
```
[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply productivity-suite/piercing-library/src/piercing-gateway.ts:6]: ah sorry I thought it was left here by mistake, I don't mind putting it back, maybe I can add a comment?

or could we just replace it with something like:
```ts
// for debugging replace `qwikloader.js` with `qwikloader.debug.js` to have the code non-minified 
```
?

File: productivity-suite/piercing-library/src/piercing-gateway.ts (hunk 1 above)
Comment (dario-piotrowicz) [reply productivity-suite/piercing-library/src/piercing-gateway.ts:109]: I've put the comment back 🙂

**Review by petebacondarwin — APPROVED** [review]

**Pull request merged**